src/app/static/src/app/components/regionPage.vue

```diff
@@ -69,6 +69,7 @@
         watch: {
             $route(to) {
                 this.setCurrentRegion(to.path);
+                this.currentStep = 1;
```

Comment:
This means that when a new region is added, the page navigates back to the baseline step. But would probably be nicer to store the current step in the region object itself so the app can remember what you were looking at on a per region basis. Out of scope here but will add ticket to backlog.